File: src/main/java/com/wayne/Hello.java
package com.wayne;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World!");
//        new Person().hello();
        Person person = new Person();
        person.hello();
        person.weight = 66.5f;
        person.height = 1.7f;
        System.out.println(person.bmi());
    }
}

File: src/main/kotlin/com/wayne/Hello.kt
package com.wayne

fun main(args: Array<String>) {
//    println("Hello Kotlin!");
//    Human().hello();
    val human = Human(66.5f, 1.7f);
    human.hello();
    println(human.bmi());
}

class Human(var weight:Float, var height:Float){
    fun bmi() : Float {
        val bmi = weight / (height*height);
        return bmi;
    }

    fun hello(){
        println("Hello Kotlin World!");
    }
}